v1/graphoid.yaml

```diff
@@ -34,7 +34,7 @@ paths:
         '200':
           description: The PNG render of the requested graph.
           schema:
-            type: blob
+            type: string
```

Comment:
PNGs are binary blobs, not strings, so we need `type: file` here (mental note: we'll need to change this to be v3-compliant)

Reply:
Done